recipes/centos_wildfly_swarm/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2012-2017 Red Hat, Inc
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+
+FROM registry.centos.org/centos/centos
+
+MAINTAINER Dharmit Shah
+
+ARG JAVA_VERSION=1.8.0
+ARG SWARM_VERSION=2017.3.2
```

Comment:
@dharmit Not super familiar with stuff. Does this version need to be updated on every WF Swarm release?